### PR TITLE
Fixes #48: Creating demo.css file

### LINKED ISSUE #48: Create a demo.css file
Create a demo.css file so that we can put demo css into it. This file should be for showcase purposes only and not get distributed via bower.

### PR DESCRIPTION
**Pull request by erundle**
Cleaning up demo css

### CODE REVIEW

File: src/views/datalist/data-list-directive.js [hunk 1]
```diff
@@ -28,16 +28,6 @@
...
-      hr {
-        display: block;
-        height: 10px;
-        border: 0;
-        border-top: 1px solid #525252;
-        margin: 1em 0;
-        padding: 0;
-      }
-    </style>
     <div ng-controller="ViewCtrl" class="row" style="display:inline-block; width: 100%;">
```
Comment (dtaylor113) [inline src/views/datalist/data-list-directive.js]: This HR style is also used in the donut, sparkline, and utilization charts

**Comment by dtaylor113** [comment]
LGTM.  Would be great if we could figure out how to get all the styles to show up when one clicks on 'Edit in Plunkr'

File: src/views/datalist/data-list-directive.js (hunk 1 above)
Comment (waldenraines) [inline src/views/datalist/data-list-directive.js]: > This HR style is also used in the donut, sparkline, and utilization charts

i.e. it needs to be removed from there also.

**Comment by waldenraines** [comment]
> Would be great if we could figure out how to get all the styles to show up when one clicks on 'Edit in Plunkr'

If we were to put the styles into their own `<file>` section they would.  The downside of that obviously is that you end up having to repeat similar styles across directives.  Part of me prefers this even though it violated DRY.

File: src/views/datalist/data-list-directive.js (hunk 1 above)
Comment (erundle) [inline src/views/datalist/data-list-directive.js]: Done

**Comment by jeff-phillips-18** [comment]
LGTM

**Comment by waldenraines** [comment]
> If we were to put the styles into their own <file> section they would. The downside of that obviously is that you end up having to repeat similar styles across directives. Part of me prefers this even though it violates DRY.

@erundle thoughts on this?

**Comment by erundle** [comment]
I personally hate style tags since I feel like it promotes bad practices which is why I created the CSS file but if we don't want to take this route I am OK with it.

**Comment by waldenraines** [comment]
> I personally hate style tags since I feel like it promotes bad practices which is why I created the CSS file but if we don't want to take this route I am OK with it.

No, I mean putting the css for each component in `<file>` tags as normal css so they are displayed in plunkr.

**Comment by dtaylor113** [comment]
I don't think it is worth re-creating all those styles in <file> just for plunkr.  Everything works in plunkr, it just isn't styled correctly.

**Comment by waldenraines** [comment]
> I don't think it is worth re-creating all those styles in <file> just for plunkr. Everything works in plunkr, it just isn't styled correctly.

Yeah, you're probably right that DRY > styles in plunkr.  I created an issue for this (#65) in case we want to try to get these styles in plunk in the future.

@erundle can you prepend "Fixes #48: " to your commit message so the issue will close upon merge and so we have a link between the two?

**Comment by erundle** [comment]
Commit message updated.

**Pull request merged**